arkouda_workflows/deploy-arkouda-on-kubernetes-workflow-template.yaml

```diff
@@ -201,6 +202,54 @@ spec:
                       - key: script
                         path: start-arkouda-locale.sh
 
+  - name: verify-locale-deployment
+    volumes:
+      - name: sa
+        secret:
+          secretName: "{{ workflow.parameters.arkouda-serviceaccount-name }}"
+    script:
+      image: bearsrus/arkouda-integration:{{ workflow.parameters.arkouda-release-version }}
+      imagePullPolicy: "{{ workflow.parameters.image-pull-policy }}" 
+      command: [python3]
+      env:
+        - name: APP_NAME
+          value: "{{ workflow.parameters.arkouda-instance-name }}-locale"
+        - name: NUM_LOCALES
+          value: "{{ workflow.parameters.arkouda-number-of-locales }}"
+        - name: K8S_HOST
+          value: "{{ workflow.parameters.kubernetes-api-url }}"
+        - name: NAMESPACE
+          value: "{{ workflow.parameters.arkouda-namespace }}"
+      volumeMounts:
+        - name: sa
+          mountPath: /opt/sa 
+      source: |
+        import os
+        import time
+        from arkouda_integration.k8s import KubernetesDao
+
+        app_name = os.environ['APP_NAME']
+        namespace = os.environ['NAMESPACE']
+        num_locales = int(os.environ['NUM_LOCALES'])
+        k8s_host = os.environ['K8S_HOST']
+
+        cacert_file = '/opt/sa/ca.crt'
+        with open("/opt/sa/token", "r") as token_file:
+            token = token_file.read()
+
+        dao = KubernetesDao(cacert_file=cacert_file,
+                            token=token,
+                            k8s_host=k8s_host)
+
+        num_deployed_locales = len(dao.get_pod_ips(namespace=namespace, app_name=app_name))
+
+        while num_deployed_locales < num_locales:
+            print(f'{num_deployed_locales} of {num_locales} locales deployed')
+            time.sleep(5)
+            num_deployed_locales = len(dao.get_pod_ips(namespace=namespace, app_name=app_name))
+
+        print(f"{num_locales} locales deployed, arkouda-locale deployment complete")
```

Comment:
I think this all looks good, just wanted to check if this print statement is needed

Reply:
yes, I need to check why it's still a print statement b/c I changed it to a python logger